pkg/controller/config/processexcluder/processexcluder.go

```diff
@@ -0,0 +1,94 @@
+package processexcluder
+
+import (
+	"sync"
+
+	configv1alpha1 "github.com/open-policy-agent/gatekeeper/apis/config/v1alpha1"
+)
+
+type Process string
+
+const (
+	Audit   = Process("audit")
+	Sync    = Process("sync")
+	Webhook = Process("webhook")
+	Star    = Process("*")
+)
+
+type Set struct {
```

Comment:
Is there some more descriptive name we could use here?

`Excluder` would stutter, unless we rename the package `process`, then it reads as `process.Excluder`

We could also just find a different name.